src/sounds.cpp

```diff
@@ -21,7 +21,7 @@
 #ifdef SDL_SOUND
 #include "SDL2/SDL_mixer.h"
 #include <thread>
-#if (defined _WIN32 || defined WINDOWS)
+#if (!defined __MINGW64__ && defined _WIN32 || defined WINDOWS )
```

Comment:
Is this really what you want or are there some parenthesis missing? Currently it means `(!a && b) || c`, but it should probably be `!a && (b || c)`.